python/dronin/logfs.py

```diff
@@ -3,7 +3,13 @@
 
 from struct import Struct
 
-config_arena_magic = 0x3bb141cf
+known_magics = {
```

Comment:
Maybe so if you took your SDCard from one board to a different type it would be erased? I suspect SD predates the magic words though...